### PR TITLE
Add `-` to allowed characters for inline HTML

### PR DESCRIPTION
**Pull request by neonichu**
This allows things like data attributes to work properly in inline HTML.

### CODE REVIEW

File: Source/MMHTMLParser.m [hunk 1]
```diff
@@ -248,6 +248,7 @@ - (NSRange)_parseNameWithScanner:(MMScanner *)scanner
 {
     NSMutableCharacterSet *nameSet = [NSMutableCharacterSet alphanumericCharacterSet];
     [nameSet addCharactersInString:@":"];
+    [nameSet addCharactersInString:@"-"];
```
Comment (mdiep) [inline Source/MMHTMLParser.m]: Could you add this in the previous call to `addCharactersInString:` instead of adding a second call?

**Comment by mdiep** [comment]
Thanks for the pull request! Just the one nit, otherwise things look great!

**Comment by neonichu** [comment]
:+1: — updated the PR.

**Comment by mdiep** [comment]
Thank you! :sparkles:

**Pull request merged**